src/opm/parser/eclipse/EclipseState/Eclipse3DProperties.cpp

```diff
@@ -443,6 +443,10 @@ namespace Opm {
         supportedDoubleKeywords.emplace_back( "THCGAS", 0.0, "Energy/AbsoluteTemperature*Length*Time", true);
         supportedDoubleKeywords.emplace_back( "THCWATER", 0.0, "Energy/AbsoluteTemperature*Length*Time", true);
 
+        // free flow boundary conditions
+        for( const auto& kw : { "FREEBCX", "FREEBCY", "FREEBCZ", "FREEBCX-", "FREEBCY-", "FREEBCZ-" } )
+            supportedDoubleKeywords.emplace_back( kw, 1.0, "1", true );
```

Comment:
the second argument should be 0 here, or else something like

```
FREEBCX
1000* /
```

will result in an open boundary.

Reply:
actually, it will be closed in the above example because the JSON for the keyword specifies 0, but

```
FREEBCX
/
```

should result in an open boundary.

Reply:
Thanks for spotting this.